lib/InputConfirmationModal/InputConfirmationModal.js

```diff
@@ -0,0 +1,99 @@
+import React, { useState } from 'react';
+import { node, string, bool, func, oneOfType } from 'prop-types';
+import {
+  Modal,
+  FormInput,
+  ButtonPrimaryDanger,
+  ButtonTertiary,
+  ButtonSecondary
+} from 'lib';
+
+function InputConfirmationModal({
```

Comment:
💚💚💚 Very subjective comment here, as this code is perf as is 👌

Although it is less verbose to have a single component that acts as both the intro and confirm page, I think it is potentially more difficult to read than if we had separate `<Intro />` and `<Confirm />` components.

The only reason I really pick up on this is that was always on a campaign for DRYer and less verbose code, at any cost. I think I have since realized that my mentality was wrong, and often more verbose and less DRY makes things much more readable!

So this comment is an attempt to counter a lot of the "this could by DRYer" comments that I left before 😂